docs/snapcraft-syntax.md

```diff
@@ -21,6 +21,11 @@ contain.
   The type of confinement supported by the snap. Can be either "devmode" (i.e.
   this snap doesn't support running under confinement) or "strict" (i.e. full
   confinement supported via interfaces).
+* `grade` (string)
+  This defines the quality grade of the snap. It can be either "devel" (i.e.
+  a development version of the snap, so not to be published to the "stable" or
+  "candidate" channels) or "stable" (i.e. a stable release or release
+  candidate).
```

Comment:
the _i.e. a stable or release candidate_ seems more to the point. But I am also afraid people will assume by reading this that a `grade: stable` snap cannot be released to the `beta` and `edge` channels.